Data.json

```
@@ -129,4 +129,9 @@
     "username": "divasgt",
     "avatar": "https://github.com/divasgt.png"
   }
+  {
+    "name": "Rajendra Rakha",
+    "username": "Rjndrkha",
+    "avatar": "https://github.com/rjndrkha.png"
```

Please make the below changes :point_down: 

```suggestion
    "avatar": "https://github.com/Rjndrkha.png"
```